operations/agent-flow-mixin/dashboards/cluster-node.libsonnet

```diff
@@ -0,0 +1,257 @@
+          panel.newQuery(
+            expr='1 - ( \r\n    rate(cluster_transport_tx_packets_failed_total{instance="$instance"}[$__rate_interval]) / \r\n    rate(cluster_transport_tx_packets_total{instance="$instance"}[$__rate_interval])\r\n)',
+            legendFormat='Tx success %',
+          ),
+          panel.newQuery(
+            expr='1 - ( \r\n    rate(cluster_transport_rx_packets_failed_total{instance="$instance"}[$__rate_interval]) / \r\n    rate(cluster_transport_rx_packets_total{instance="$instance"}[$__rate_interval])\r\n)',
+            legendFormat='Rx success %',
+          ),
+        ]) +
+        panel.withUnit('percentunit')
+      ),
+      // Pending packet queue
+      (
+        panel.new('Pending packet queue', 'timeseries') +
+        panel.withPosition({
+          h: 8,
+          w: 8,
+          x: 16,
+          y: 18,
+        }) +
+        panel.withQueries([
+          panel.newQuery(
+            expr='cluster_transport_tx_packet_queue_length{instance="$instance"}',
+            legendFormat='tx queue',
+          ),
+          panel.newQuery(
+            expr='cluster_transport_rx_packet_queue_length{instance="$instance"}',
+            legendFormat='rx queue',
+          ),
+        ]) +
+        panel.withUnit('pkts')
+      ),
+      // Stream bandwidth
+      (
+        panel.new('Stream bandwidth', 'timeseries') +
+        panel.withPosition({
+          h: 8,
+          w: 8,
+          x: 0,
+          y: 26,
+        }) +
+        panel.withQueries([
+          panel.newQuery(
+            expr='rate(cluster_transport_stream_rx_bytes_total{instance="$instance"}[$__rate_interval])',
+            legendFormat='rx',
+          ),
+          panel.newQuery(
+            expr='-1 * rate(cluster_transport_stream_tx_bytes_total{instance="$instance"}[$__rate_interval])',
+            legendFormat='tx',
+          ),
+        ]) +
+        panel.withUnit('Bps')
+      ),
+      // Stream write success rate
+      (
+        panel.new('Stream write success rate', 'timeseries') +
+        panel.withPosition({
+          h: 8,
+          w: 8,
+          x: 8,
+          y: 26,
+        }) +
+        panel.withQueries([
+          panel.newQuery(
+            expr='1 - ( \r\n    rate(cluster_transport_stream_tx_packets_failed_total{instance="$instance"}[$__rate_interval]) / \r\n    rate(cluster_transport_stream_tx_packets_total{instance="$instance"}[$__rate_interval])\r\n)',
+            legendFormat='Tx success %'
+          ),
+          panel.newQuery(
+            expr='1 - ( \r\n    rate(cluster_transport_stream_rx_packets_failed_total{instance="$instance"}[$__rate_interval]) / \r\n    rate(cluster_transport_stream_rx_packets_total{instance="$instance"}[$__rate_interval])\r\n)',
+            legendFormat='Rx success %'
+          ),
+        ]) +
+        panel.withUnit('percentunit')
+      ),
+      // Open transport streams
+      (
+        panel.new('Open transport streams', 'timeseries') +
```

Comment:
Ditto: can we add a description + detail around when it should be concerning?

Reply:
Looks good. I would add information around how the transport streams should eventually close, and the graph should tend towards zero. Anything else is problematic.

> continuously gossip messages

continuously gossiping messages*

Reply:
> how the transport streams should eventually close

Is that true? My 6-node cluster has been stable for a couple of days and each node has 3-5 open streams at any time. Do you mean during cluster changes?

Reply:
I think it's supposed to be true 😬 I thought transport streams should only open for the streaming API calls, and should close after the net.Conn is closed by memberlist?

(And transport streams are only opened for a push/pull, which happens once a minute with a random node)

Reply:
Whoops, will have to check. Not sure if it could be due to the timeout hapenning on a similar timeframe? :/